### PR TITLE
implement ability to generate sitemap.xml file

### PR DESCRIPTION
**Pull request by ihor-bobyr**
took config from here - https://programming.org.ua/sitemap.xml

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
...
+          'contacts',
+          'email-confirmed',
+          'feedback-all',
+          '',
+          'support',
+          'courses/adults',
+          'courses',
+          'forms/feedback-form',
+          'forms/first-form',
+          'forms/last-form',
```
Comment (ihor-bobyr) [inline webpack.config.js:104]: remove

File: webpack.config.js [hunk 2]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
...
+          'forms/main-form',
```
Comment (ihor-bobyr) [inline webpack.config.js:105]: remove

File: webpack.config.js [hunk 3]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
...
+          'forms/main-school-form',
```
Comment (ihor-bobyr) [inline webpack.config.js:106]: remove

File: webpack.config.js [hunk 4]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
...
+        ...[
+          'about',
+          'contacts',
+          'email-confirmed',
+          'feedback-all',
+          '',
+          'support',
+          'courses/adults',
+          'courses',
+          'forms/feedback-form',
```
Comment (ihor-bobyr) [inline webpack.config.js:102]: update route

File: webpack.config.js [hunk 5]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
...
+          'forms/first-form',
```
Comment (ihor-bobyr) [inline webpack.config.js:103]: update route

File: webpack.config.js [hunk 6]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
     clean: true,
   },
   plugins: [
+    new SitemapPlugin({
+      base: BASE_URL,
+      paths: [
+        ...[
+          'about',
+          'contacts',
+          'email-confirmed',
```
Comment (ihor-bobyr) [inline webpack.config.js:96]: remove

File: webpack.config.js [hunk 7]
```diff
@@ -86,6 +87,35 @@ module.exports = async (_, { mode = 'development' }) => ({
...
+          'feedback-all',
+          '',
+          'support',
```
Comment (ihor-bobyr) [inline webpack.config.js:99]: update to supportus

**Comment by k-boiko** [comment]
Regarding array of locales, it seems that your current code should be perfectly fine, according to this article
https://sajidhasan054.medium.com/hreflang-implementation-unsing-xml-sitemap-616afb4f6a41

File: webpack.config.js (hunk 1 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

File: webpack.config.js (hunk 2 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

File: webpack.config.js (hunk 3 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

File: webpack.config.js (hunk 4 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

File: webpack.config.js (hunk 5 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

File: webpack.config.js (hunk 6 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

File: webpack.config.js (hunk 7 above)
Comment (ihor-bobyr) [reply webpack.config.js]: done

**Pull request merged**